src/main/java/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,53 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Ladder {
+
+    private List<Participant> participantList;
+    private List<Line> lineList = new ArrayList<>();
```

Comment:
인스턴스 변수 이름에 직접적인 구현체를 드러내기보단, 적당한 의미를 담을 수 있게 이름을 지어주는 것이 좋을 것 같습니다 :)

```suggestion
    private List<Participant> participants;
    private List<Line> lines = new ArrayList<>();
```